executor/runtime_work/codex_discovery.py

```
@@ -0,0 +1,468 @@
+        parsed = _parse_datetime(value.strip())
+        return parsed.isoformat() if parsed else value.strip()
+    return None
+
+
+def _codex_enum_value(enum_name: str, value: str) -> Any:
+    try:
+        from openai_codex.generated import v2_all
+
+        enum_type = getattr(v2_all, enum_name)
+        return enum_type(value)
+    except Exception:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)